schema-registry/src/test/scala/net/manub/embeddedkafka/schemaregistry/EmbeddedKafkaWithSchemaRegistrySpec.scala

```diff
@@ -1,28 +1,27 @@
-package net.manub.embeddedkafka
-
-import avro.schemaregistry._
+package net.manub.embeddedkafka.schemaregistry
 
 import net.manub.embeddedkafka.Codecs._
+import net.manub.embeddedkafka.{EmbeddedKafkaSpecSupport, TestAvroClass}
 import org.apache.kafka.clients.producer.ProducerRecord
 
 import scala.collection.JavaConverters._
 
-class EmbeddedKafkaSchemaRegistrySpec
+class EmbeddedKafkaWithSchemaRegistrySpec
```

Comment:
Could we drop the "WithSchemaRegistry" part since this class is already on a different package?